### PR TITLE
fix(security): Describe `occ security:bruteforce:attempts` accurately

### PR DESCRIPTION
**Pull request by joshtrichards**
## Summary

This command is used to list a summary of brute force protection attempts associated with a given IP address and, optionally, a specific action. It does not reset attempts (there's already another command for that).

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by solracsf — APPROVED** [review]

File: core/Command/Security/BruteforceAttempts.php [hunk 1]
```diff
@@ -41,11 +41,11 @@ protected function configure(): void {
 		parent::configure();
 		$this
 			->setName('security:bruteforce:attempts')
-			->setDescription('resets bruteforce attempts for given IP address')
+			->setDescription('list bruteforce attempts for given IP address')
```
Comment (nickvergessen) [inline core/Command/Security/BruteforceAttempts.php:44]:
```suggestion
			->setDescription('List bruteforce attempts for given IP address')
```

But it's not actually a list. More like the statistics. So maybe something like:

```suggestion
			->setDescription('Bruteforce attempts information for given IP address')
```

Comment (joshtrichards) [reply core/Command/Security/BruteforceAttempts.php:44]: Agreed. "list" came from the existing Admin Manual. (I'll update the final choice over there when this is done).
```suggestion
			->setDescription('show bruteforce attempts status for a given IP address')
```

File: core/Command/Security/BruteforceAttempts.php [hunk 2]
```diff
@@ -41,11 +41,11 @@ protected function configure(): void {
 		parent::configure();
 		$this
 			->setName('security:bruteforce:attempts')
-			->setDescription('resets bruteforce attempts for given IP address')
+			->setDescription('show bruteforce attempts status for a given IP address')
 			->addArgument(
 				'ipaddress',
 				InputArgument::REQUIRED,
-				'IP address for which the attempts are to be reset',
+				'IP address for which the attempts are to be listed',
```
Comment (joshtrichards) [inline core/Command/Security/BruteforceAttempts.php:48]:
```suggestion
				'IP address for which the attempts status is to be shown',
```

File: core/Command/Security/BruteforceAttempts.php [hunk 3]
```diff
@@ -41,11 +41,11 @@ protected function configure(): void {
 		parent::configure();
 		$this
 			->setName('security:bruteforce:attempts')
-			->setDescription('resets bruteforce attempts for given IP address')
+			->setDescription('show bruteforce attempts status for a given IP address')
```
Comment (nickvergessen) [inline core/Command/Security/BruteforceAttempts.php:44]:
```suggestion
			->setDescription('Show bruteforce attempts status for a given IP address')
```

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**